### PR TITLE
Automate cubedash db update on eks stack

### PR DESCRIPTION
**Pull request by santoshamohan**
Following changes to `datacube-dashboard` chart:
1) Update chart version
2) Re-align spacing for `terminationGracePeriodSeconds` template variable
3) Update annotations and repository name in `values.yaml` file
4) Update metadata names in the template files

### CODE REVIEW

File: stable/datacube-dashboard/values.yaml [hunk 1]
```diff
@@ -4,7 +4,7 @@
 replicaCount: 1
 image:
   registry: docker.io
-  repository: opendatacube/dashboard
+  repository: opendatacube/datacube-explorer
```
Comment (harshurampur) [inline stable/datacube-dashboard/values.yaml]: There is no docker `datacube-explorer` it should still be `dashboard`

File: stable/datacube-dashboard/values.yaml [hunk 2]
```diff
@@ -21,15 +21,15 @@ resources:
   limits:
     cpu: "300m"
     memory: 1024Mi
-dockerArgs: [ "gunicorn", "-b", "0.0.0.0:8080", "-w", "1", "--timeout", "60", "cubedash:app" ]
+dockerArgs: [ "gunicorn", "-b", "0.0.0.0:8080", "-w", "4", "--timeout", "60", "cubedash:app" ]
 annotations:
-  iam.amazonaws.com/role: kubernetes-wms
+  iam.amazonaws.com/role: user.dev-eks-datacube
```
Comment (harshurampur) [inline stable/datacube-dashboard/values.yaml]: You cannot set the user specific IAM roles check with the wms roles for EKS cluster

File: stable/datacube-dashboard/values.yaml [hunk 3]
```diff
@@ -21,15 +21,15 @@ resources:
...
 additionalEnvironmentVars: { }
 update:
-  enabled: false
+  enabled: true
```
Comment (harshurampur) [inline stable/datacube-dashboard/values.yaml]: Should not enable cronjob by default

File: stable/datacube-dashboard/templates/deployment.yaml [hunk 4]
```diff
@@ -62,7 +62,6 @@ spec:
         resources:
 {{ toYaml .Values.resources | indent 12 }}
         args: [ {{- range .Values.dockerArgs }} {{ . | quote }}, {{ end -}} ]
-        terminationGracePeriodSeconds: 30
```
Comment (harshurampur) [inline stable/datacube-dashboard/templates/deployment.yaml]: This should be set as if the service cannot find use cases the job should be terminated

File: stable/datacube-dashboard/values.yaml (hunk 2 above)
Comment (santoshamohan) [reply stable/datacube-dashboard/values.yaml]: Created a new IMA role `dev-eks-datacube-cubedash`

File: stable/datacube-dashboard/values.yaml (hunk 1 above)
Comment (santoshamohan) [reply stable/datacube-dashboard/values.yaml]: Updated to `dashboard` docker name

File: stable/datacube-dashboard/values.yaml (hunk 3 above)
Comment (santoshamohan) [reply stable/datacube-dashboard/values.yaml]: `Cronjob` disabled in the template

File: stable/datacube-dashboard/templates/deployment.yaml (hunk 4 above)
Comment (santoshamohan) [reply stable/datacube-dashboard/templates/deployment.yaml]: Added `terminationGracePeriodSeconds`

File: stable/datacube-dashboard/values.yaml [hunk 5]
```diff
@@ -4,7 +4,7 @@
 replicaCount: 1
 image:
   registry: docker.io
-  repository: opendatacube/datacube-explorer
+  repository: opendatacube/datacube
```
Comment (harshurampur) [inline stable/datacube-dashboard/values.yaml]: it should be `dashboard`

File: stable/datacube-dashboard/values.yaml [hunk 6]
```diff
@@ -23,10 +23,10 @@ resources:
     memory: 1024Mi
 dockerArgs: [ "gunicorn", "-b", "0.0.0.0:8080", "-w", "4", "--timeout", "60", "cubedash:app" ]
 annotations:
-  iam.amazonaws.com/role: user.dev-eks-datacube
+  iam.amazonaws.com/role: dev-eks-datacube-cubedash
```
Comment (harshurampur) [inline stable/datacube-dashboard/values.yaml]: No need to create a new one you can use existing role for wms

Comment (santoshamohan) [reply stable/datacube-dashboard/values.yaml]: Existing role does not have sufficient permission to use `EKS`/`S3`/`RDS`/`CloudWatchEvents` resource

**Pull request merged**